architecture-decision-records/0035-metadata-csv-utf-8-file-validation.md

```diff
@@ -0,0 +1,30 @@
+# 35. Metadata CSV file UTF-8 Validation
+
+**Date**: 2024-10-07
+
+## Context
+
+Digital assets sent to The National Archives (TNA) are transferred with associated metadata in CSV format. The entry point for digital assets to TNA will be through the services provided by the Digital Selection and Transfer team.
+
+The structure and integrity of the CSV file is checked before the actual [data is validated](0034-metadata-validation.md) 
+
+## Structure and integrity checks
+
+1. **Virus check**: All files uploaded to TDR are checked for viruses using the virus check lambda
+2. **[UTF-8](utf-8-validation)**: The CSV file must be UTF-8 format
+
```

Comment:
Can you structure this as per other ADRs, ie:

**Context**
Cover the scenario, ie need to check that the ingested metadata file is encoded as UTF-8 to ensure data integrity for downstream services.
**Decision**
Describe the decision to use BOM and the reason why. Along with the implications, ie limitations with Libre office / Macs
**Other Options Considered**
Cover other possible options and why they were rejected.

Reply:
removed and changed structure